bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -526,6 +541,13 @@ pub struct ImageInfo {
     pub blurhash: Option<String>,
 }
 
+#[derive(Clone, uniffi::Record)]
+pub struct AudioInfo {
+    // FIXME: duration should be a std::time::Duration once the UniFFI proc-macro API adds support for that
```

Comment:
```suggestion
    // FIXME: duration should be a std::time::Duration once the UniFFI proc-macro API adds support
    // for that
```

Reply:
Thanks for the quick review! 
I just added a missing downcast (since as_millis() returns a u128, which then causes problems).